Rakefile

```diff
@@ -23,9 +23,14 @@ task :build do
   e "go build -v ./..."
 end
 
-desc 'Test all projects'
+desc 'Test all projects (short only)'
 task :test => [:build] do
-  e "go test -timeout 10s ./..."
+  e "go test -short -timeout 10s ./..."
+end
+
+desc 'Test all projects'
+task :test_all => [:build] do
+  e "go test -short -timeout 120s ./..."
```

Comment:
the tests that use `github.com/hashicorp/consul/testutil` are very slow, ~5s each. 

I was thinking that local developers would still use `rake test` and have fast tests and travis would run the full suite with long tests included, happy to take any suggestions